be/src/storage/version_graph.h

```diff
@@ -80,8 +80,8 @@ class VersionGraph {
     // version.end_version + 1.
     // Use adjacency list to describe version graph.
     std::unordered_map<int64_t, std::unique_ptr<Vertex>> _version_graph;
-    // max continuous version from 0
-    int64_t _max_continuous_version{0};
+    // max continuous version from -1
+    int64_t _max_continuous_version{-1};
```

Comment:
if you change here, will `if (task.version > 0) {` should be changed too?
This is in the publish task?

Reply:
why? This is for the case that after creating tablet, will add an init 0-1 rowset, then -1 will become 1, it's not related to publish task